scripts/release_info.py

```diff
@@ -292,24 +303,111 @@ def _check_storage_models(current_release):
     return [item for item in diff_list if item.startswith('core/storage')]
 
 
+def check_blocking_bugs(repo):
```

Comment:
will it be possible to do this without passing the whole object?

Should we create a `GithubRepo` class which will handel all this method?

Reply:
Yes, we can do that. We can also use a global variable instead. Which do you suggest?